doc/aggregation.md

```diff
@@ -0,0 +1,164 @@
+# Enable ClickHouse aggregation
+
+It's enabled by default since [#157](https://github.com/lomik/graphite-clickhouse/pull/157). You can disable it to use aggregation in graphite-clickhouse.
```

Comment:
I would mention version or commit hash here, as user who is not following this repo might get confused by that.

It would be a good idea to mention pros and cons for both aggregation options at the beginning.

Reply:
I added the version. Will tag the new version soon for sure